suite-native/app/package.json

```diff
@@ -59,7 +61,9 @@
         "react-native-safe-area-context": "^4.3.1",
         "react-native-screens": "^3.13.1",
         "react-native-splash-screen": "^3.3.0",
+        "react-native-svg": "^12.4.4",
         "react-native-vision-camera": "^2.14.1",
+        "react-qr-code": "^2.0.8",
```

Comment:
This package should be enough, but for future we should consider creating our own using Skia + `qr.js`, becuase it could be done in like 50 lines of code and we will get rid this depednency on whole react-native-svg + react-qr-code. If you will check source `react-qr-code` it's basically few lines of code and you can replace svg components with Skia components and that's it.

But for now, this is something for backlog with low priority.